atomic_reactor/plugins/post_compress.py

```diff
@@ -85,5 +88,12 @@ def run(self):
             self.log.info('fetching image %s from docker', image)
             with self.tasker.d.get_image(image) as image_stream:
                 outfile = self._compress_image_stream(image_stream)
-        self.workflow.exported_image_sequence.append(get_exported_image_metadata(outfile))
+        metadata = get_exported_image_metadata(outfile)
+        self.workflow.exported_image_sequence.append(metadata)
+
+        compressed_size = metadata['size']
+        if compressed_size != 0:
+            self.log.debug('uncompressed: %s, compressed: %s, ratio: %.2f',
+                human_size(self.uncompressed_size), human_size(compressed_size),
+                self.uncompressed_size/float(compressed_size))
```

Comment:
Nice! Does this ratio calculation agree with what `gzip -l` says though? I think it should be size_decrease / uncompressed_size.